examples/terminal.py

```diff
@@ -23,6 +23,26 @@
 
 import urwid
 
+from urwid import raw_display, escape
+
+class BracketedPasteScreen(urwid.raw_display.Screen):
```

Comment:
I re-checked `Screen` code and IMHO we can extend it for bracketed paste support directly